pilot/pkg/model/sidecar.go

```diff
@@ -97,8 +98,7 @@ type SidecarScope struct {
 	// corresponds to a service in the services array above. When computing
 	// CDS, we simply have to find the matching service and return the
 	// destination rule.
-	destinationRules map[host.Name]*config.Config
-
+	destinationRules map[host.Name][]*config.Config
```

Comment:
type map[host.Name][namespace]*config.Config ?

Reply:
I have another question: if a DR without workload selector, and a DR with selector/ Both applies to a proxy. Then do they merge? Or the one with sector takes priority

Reply:
The filtering logic for the specific proxy will get a merged version of DR with selector + without selector. However other proxies in the same namespace will get the "without" selector DR.

Reply:
THis logic is right.